RNA-seq/00a-reproducibility_cmdline.md

```diff
@@ -8,33 +8,32 @@ However, it is very hard to describe exactly what was done after the fact.
 
 In contrast with GUIs, the command line interfaces that we have started to use in this set of exercises provide reproducibility.
 But each step can require extensive manual entry.
-As we get to know our command line, we can get reproducibility and efficiency.
+As we get to know our command line, we can increase reproducibility and efficiency.
 We can also script commands so that we can run them again anytime.
 
-#### Shell scripts
+### Shell scripts
 
 When you are running commands by typing them into a command line, they are being interpreted and run by a "shell" (think of this as the program that converts your typed commands into steps that the computer runs).
-We've been typing things one-by-one.
-We can, instead, put many of these commands together into a single file.
+We could copy and paste commands into Terminal one-by-one.
+Alternatively, we can put many of these commands together into a single file.
 This is something called a "shell script."
 This is the preferred way to interact with a command line, because it provides a record of the exact set of commands that were run.
+We used a shell script (`scripts/run_SRR585570.sh`) in the first notebook (`01-qc_trim_quant.md`) to execute the FastQC, fastp, and Salmon steps for a single sample sequentially.
 
-Let's create our first shell script.
-In RStudio go to the file menu, select New File, Text File.
-Name the script `rnaseq.sh` and put it in the kitematic folder.
+To create a new shell script: in RStudio go to the file menu, select New File, Text File.
```

Comment:
```suggestion
To create a new shell script: in RStudio go to the file menu > select New File >Text File.
```
Are these `>` appropriate here??

Reply:
Yes I like the arrows, I'll also put backticks around `File`, etc.